prefect_snowflake/database.py

```diff
@@ -42,9 +42,7 @@ class SnowflakeConnector(Block):
     schema_: str = Field(
         alias="schema", description="The name of the default schema to use"
     )
-    credentials: SnowflakeCredentials = Field(
-        ..., description="The credentials to authenticate with Snowflake"
-    )
+    credentials: SnowflakeCredentials
```

Comment:
Removing the field is a work around for some UI behavior, right?

Reply:
Yes, with Field(), it yields a JSON fields.